apps/flows/src/app/transaction-details-flow/for-you/for-you.component.scss

```diff
@@ -2,3 +2,71 @@
   position: absolute;
   inset: 0;
 }
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-gap: 24px;
+  gap: 24px;
+}
+
+/* Let grid items span all columns by default */
+.grid-item {
+  grid-column: span 12;
+}
+
+/* Tablet size and above */
+@media (min-width: 721px) {
```

Comment:
You should use the `include-media` Sass utility and our size design tokens instead of hardcoded values for breakpoints.

You can also utilize Sass variables for column span and gap sizes.

See examples in `apps/cookbook/src/app/examples/grid-layout-example/grid-layout-extended-example/grid-layout-extended-example.component.scss` and `apps/cookbook/src/app/examples/grid-layout-example/grid-layout-single-container-example/grid-layout-single-container-example.component.scss`

Reply:
I checked the examples and include-media and size design tokens are used instead